util/cookies.ts

```diff
@@ -0,0 +1,33 @@
+import { serialize } from 'cookie';
+
+export function createSerializedRegisterSessionTokenCookie(token: string) {
+  // Use secure cookies in production (HTTPS only)
+  const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
We can inline this below

(also same feedback for the other function `deleteSerializedRegisterSessionTokenCookie` below)

Reply:
[5a205ab](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/5a205abbb49c3eb76982f4177b159bfbca6edb9f)